libsql-storage/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "libsql-storage"
+version = "0.0.1"
+edition = "2021"
+description = "libSQL WAL"
+repository = "https://github.com/tursodatabase/libsql"
+license = "MIT"
+
+[dependencies]
+prost = "0.12"
+tonic = { version = "0.10.0", features = ["tls"] }
+
+[dev-dependencies]
+prost-build = "0.12.0"
```

Comment:
nit: should just drop the trailing `.0` since that is the format we have elsewhere in this